payloads/library/Incident_Response/Hidden_Images/payload.txt

```diff
@@ -1,12 +1,7 @@
-
-
-
-LED R B 100
+LED SETUP
+GET SWITCH_POSITION
 ATTACKMODE HID STORAGE
 
-
-DUCKY_LANG gb
-LED B
+LED ATTACK
 RUN WIN powershell -executionpolicy Bypass ".((gwmi win32_volume -f 'label=''BashBunny''').Name+'payloads\\$SWITCH_POSITION\run.ps1')"
-LED G FAST
-#Green means good to go
+LED FINISH

```

Comment:
This should probably be LED ATTACK or similar.

Reply:
@sebkinne thank you for reviewing my work. I will make the changes right now.